code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5273,6 +5273,32 @@
 					H.adjustToxLoss(3)
 				H.adjustToxLoss(0.3)
 
+/datum/reagent/ethanol/deadrum/bonejuice
+	name = "Bone Hurting Juice"
+	id = BONEJUICE
+	description = "A juice that hurts your bones. Possibly addictive."
+	reagent_state = LIQUID
+	color = "#8E18A9" //rgb: 142, 24, 169
+
+/datum/reagent/ethanol/deadrum/bonejuice/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.species && (H.species.anatomy_flags & NO_BONES))
+			return
+		else
+			switch(volume)
+				if(1 to 50)
+					if(prob(15))
+						to_chat(H,"<span class='warning'>Ow. Oof. Oooouch. Your bones hurt.</span>")
```

Comment:
This just seems to have been made for the memes. What does this contribute to the game, at all?

It doesn't even do anything for 50 units, and that's a fucking beaker full